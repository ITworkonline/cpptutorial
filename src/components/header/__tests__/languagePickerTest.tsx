import React from "react";
import renderer from "react-test-renderer";
import LanguagePicker from "../languagePicker";
import { ThemeProvider } from "emotion-theming";
import { lightTheme } from "../../theme";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={lightTheme}>
          <LanguagePicker />
        </ThemeProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
