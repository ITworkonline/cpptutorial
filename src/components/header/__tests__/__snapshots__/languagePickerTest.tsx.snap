// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="css-1b54bo5-StyledLanguagePicker eeyb89v0"
>
  <svg
    id="Capa_1"
    version="1.1"
    viewBox="0 0 469.333 469.333"
    x="0px"
    xmlns="http://www.w3.org/2000/svg"
    y="0px"
  >
    <g>
      <g>
        <g>
          <path
            d="M253.227,300.267L253.227,300.267L199.04,246.72l0.64-0.64c37.12-41.387,63.573-88.96,79.147-139.307h62.507V64H192 V21.333h-42.667V64H0v42.453h238.293c-14.4,41.173-36.907,80.213-67.627,114.347c-19.84-22.08-36.267-46.08-49.28-71.467H78.72 c15.573,34.773,36.907,67.627,63.573,97.28l-108.48,107.2L64,384l106.667-106.667l66.347,66.347L253.227,300.267z"
          />
          <path
            d="M373.333,192h-42.667l-96,256h42.667l24-64h101.333l24,64h42.667L373.333,192z M317.333,341.333L352,248.853 l34.667,92.48H317.333z"
          />
        </g>
      </g>
    </g>
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
    <g />
  </svg>
  <ul>
    <li
      css={
        Object {
          "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImxhbmd1YWdlUGlja2VyLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUF5Q21CIiwiZmlsZSI6Imxhbmd1YWdlUGlja2VyLnRzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tIFwicmVhY3RcIjtcbmltcG9ydCB7IGNzcywganN4IH0gZnJvbSBcIkBlbW90aW9uL2NvcmVcIjtcbmltcG9ydCBzdHlsZWQgZnJvbSBcIkBlbW90aW9uL3N0eWxlZFwiO1xuaW1wb3J0IHsgdXNlVGhlbWUgfSBmcm9tIFwiZW1vdGlvbi10aGVtaW5nXCI7XG5pbXBvcnQgTGluayBmcm9tIFwiZ2F0c2J5LWxpbmtcIjtcblxuaW1wb3J0IHsgVGhlbWUgfSBmcm9tIFwiLi4vdGhlbWVcIjtcblxuaW1wb3J0IHsgTGFuZ3VhZ2UsIGxhbmd1YWdlcywgdHJhbnNsYXRpb25zIH0gZnJvbSBcIi4uLy4uL3V0aWxzL3RyYW5zbGF0aW9uc1wiO1xuXG5pbXBvcnQgTGFuZ3VhZ2VTd2l0Y2hJY29uIGZyb20gXCIuLi9pbWFnZXMvbGFuZ3VhZ2Utc3dpdGNoXCI7XG5cbmNvbnN0IFN0eWxlZExhbmd1YWdlUGlja2VyID0gc3R5bGVkLmRpdjx7IHRoZW1lOiBUaGVtZSB9PmBcbiAgc3ZnIHtcbiAgICBmaWxsOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5pY29ufTtcbiAgICB3aWR0aDogMzBweDtcbiAgfVxuXG4gIHN2Zzpob3ZlciB7XG4gICAgZmlsbDogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaWNvbkhvdmVyfTtcbiAgfVxuXG4gIHVsIHtcbiAgICBkaXNwbGF5OiBub25lO1xuICAgIHBvc2l0aW9uOiBhYnNvbHV0ZTtcbiAgfVxuXG4gIDpob3ZlciA+IHVsIHtcbiAgICBkaXNwbGF5OiBmbGV4O1xuICAgIGZsZXgtZGlyZWN0aW9uOiBjb2x1bW47XG4gICAgYmFja2dyb3VuZC1jb2xvcjogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaGVhZGVyQmd9O1xuICB9XG5gO1xuXG5pbnRlcmZhY2UgTGFuZ3VhZ2VPcHRpb25Qcm9wcyB7XG4gIGxhbmc6IExhbmd1YWdlO1xuICBhY3RpdmU6IGJvb2xlYW47XG59XG5cbmNvbnN0IExhbmd1YWdlT3B0aW9uID0gKHsgbGFuZywgYWN0aXZlIH06IExhbmd1YWdlT3B0aW9uUHJvcHMpID0+IHtcbiAgY29uc3QgdGhlbWU6IFRoZW1lID0gdXNlVGhlbWUoKTtcbiAgY29uc3Qgc3R5bGUgPSBjc3NgXG4gICAgcGFkZGluZzogMTBweCAxNXB4O1xuXG4gICAgJjpob3ZlciB7XG4gICAgICBiYWNrZ3JvdW5kOiAke3RoZW1lLmNvbG9ycy5oaWdobGlnaHR9O1xuICAgIH1cblxuICAgIGEge1xuICAgICAgY29sb3I6ICR7dGhlbWUuY29sb3JzLnByaW1hcnlUZXh0fTtcbiAgICAgIGZvbnQtc2l6ZTogMTZweDtcbiAgICAgIGZvbnQtd2VpZ2h0OiA1MDA7XG4gICAgICBsaW5lLWhlaWdodDogMWVtO1xuICAgICAgb3BhY2l0eTogMTtcbiAgICB9XG4gIGA7XG5cbiAgY29uc3QgYWN0aXZlU3R5bGUgPSBjc3NgXG4gICAgZm9udC13ZWlnaHQ6IDcwMDtcbiAgYDtcblxuICByZXR1cm4gKFxuICAgIDxsaSBjc3M9e1tzdHlsZSwgYWN0aXZlICYmIGFjdGl2ZVN0eWxlXX0+XG4gICAgICA8TGluayBsYW5nPXtsYW5nfSB0bz1cIi9cIj5cbiAgICAgICAge3RyYW5zbGF0aW9uc1tsYW5nXVtcImxhbmd1YWdlX25hbWVcIl19XG4gICAgICA8L0xpbms+XG4gICAgPC9saT5cbiAgKTtcbn07XG5cbmNvbnN0IExhbmd1YWdlUGlja2VyID0gKCkgPT4ge1xuICBjb25zdCBhY3RpdmVMYW5nID0gXCJlblwiO1xuXG4gIHJldHVybiAoXG4gICAgPFN0eWxlZExhbmd1YWdlUGlja2VyPlxuICAgICAgPExhbmd1YWdlU3dpdGNoSWNvbiAvPlxuICAgICAgPHVsPlxuICAgICAgICB7bGFuZ3VhZ2VzLm1hcChsYW5nID0+IChcbiAgICAgICAgICA8TGFuZ3VhZ2VPcHRpb24ga2V5PXtsYW5nfSBsYW5nPXtsYW5nfSBhY3RpdmU9e2xhbmcgPT0gYWN0aXZlTGFuZ30gLz5cbiAgICAgICAgKSl9XG4gICAgICA8L3VsPlxuICAgIDwvU3R5bGVkTGFuZ3VhZ2VQaWNrZXI+XG4gICk7XG59O1xuXG5leHBvcnQgZGVmYXVsdCBMYW5ndWFnZVBpY2tlcjtcbiJdfQ== */",
          "name": "1rv6e8b-style-activeStyle-LanguageOption",
          "next": undefined,
          "styles": "padding:10px 15px;&:hover{background:#3884C7;}a{color:#3B454E;font-size:16px;font-weight:500;line-height:1em;opacity:1;};label:style;;font-weight:700;;label:activeStyle;;/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImxhbmd1YWdlUGlja2VyLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUF5RHlCIiwiZmlsZSI6Imxhbmd1YWdlUGlja2VyLnRzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tIFwicmVhY3RcIjtcbmltcG9ydCB7IGNzcywganN4IH0gZnJvbSBcIkBlbW90aW9uL2NvcmVcIjtcbmltcG9ydCBzdHlsZWQgZnJvbSBcIkBlbW90aW9uL3N0eWxlZFwiO1xuaW1wb3J0IHsgdXNlVGhlbWUgfSBmcm9tIFwiZW1vdGlvbi10aGVtaW5nXCI7XG5pbXBvcnQgTGluayBmcm9tIFwiZ2F0c2J5LWxpbmtcIjtcblxuaW1wb3J0IHsgVGhlbWUgfSBmcm9tIFwiLi4vdGhlbWVcIjtcblxuaW1wb3J0IHsgTGFuZ3VhZ2UsIGxhbmd1YWdlcywgdHJhbnNsYXRpb25zIH0gZnJvbSBcIi4uLy4uL3V0aWxzL3RyYW5zbGF0aW9uc1wiO1xuXG5pbXBvcnQgTGFuZ3VhZ2VTd2l0Y2hJY29uIGZyb20gXCIuLi9pbWFnZXMvbGFuZ3VhZ2Utc3dpdGNoXCI7XG5cbmNvbnN0IFN0eWxlZExhbmd1YWdlUGlja2VyID0gc3R5bGVkLmRpdjx7IHRoZW1lOiBUaGVtZSB9PmBcbiAgc3ZnIHtcbiAgICBmaWxsOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5pY29ufTtcbiAgICB3aWR0aDogMzBweDtcbiAgfVxuXG4gIHN2Zzpob3ZlciB7XG4gICAgZmlsbDogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaWNvbkhvdmVyfTtcbiAgfVxuXG4gIHVsIHtcbiAgICBkaXNwbGF5OiBub25lO1xuICAgIHBvc2l0aW9uOiBhYnNvbHV0ZTtcbiAgfVxuXG4gIDpob3ZlciA+IHVsIHtcbiAgICBkaXNwbGF5OiBmbGV4O1xuICAgIGZsZXgtZGlyZWN0aW9uOiBjb2x1bW47XG4gICAgYmFja2dyb3VuZC1jb2xvcjogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaGVhZGVyQmd9O1xuICB9XG5gO1xuXG5pbnRlcmZhY2UgTGFuZ3VhZ2VPcHRpb25Qcm9wcyB7XG4gIGxhbmc6IExhbmd1YWdlO1xuICBhY3RpdmU6IGJvb2xlYW47XG59XG5cbmNvbnN0IExhbmd1YWdlT3B0aW9uID0gKHsgbGFuZywgYWN0aXZlIH06IExhbmd1YWdlT3B0aW9uUHJvcHMpID0+IHtcbiAgY29uc3QgdGhlbWU6IFRoZW1lID0gdXNlVGhlbWUoKTtcbiAgY29uc3Qgc3R5bGUgPSBjc3NgXG4gICAgcGFkZGluZzogMTBweCAxNXB4O1xuXG4gICAgJjpob3ZlciB7XG4gICAgICBiYWNrZ3JvdW5kOiAke3RoZW1lLmNvbG9ycy5oaWdobGlnaHR9O1xuICAgIH1cblxuICAgIGEge1xuICAgICAgY29sb3I6ICR7dGhlbWUuY29sb3JzLnByaW1hcnlUZXh0fTtcbiAgICAgIGZvbnQtc2l6ZTogMTZweDtcbiAgICAgIGZvbnQtd2VpZ2h0OiA1MDA7XG4gICAgICBsaW5lLWhlaWdodDogMWVtO1xuICAgICAgb3BhY2l0eTogMTtcbiAgICB9XG4gIGA7XG5cbiAgY29uc3QgYWN0aXZlU3R5bGUgPSBjc3NgXG4gICAgZm9udC13ZWlnaHQ6IDcwMDtcbiAgYDtcblxuICByZXR1cm4gKFxuICAgIDxsaSBjc3M9e1tzdHlsZSwgYWN0aXZlICYmIGFjdGl2ZVN0eWxlXX0+XG4gICAgICA8TGluayBsYW5nPXtsYW5nfSB0bz1cIi9cIj5cbiAgICAgICAge3RyYW5zbGF0aW9uc1tsYW5nXVtcImxhbmd1YWdlX25hbWVcIl19XG4gICAgICA8L0xpbms+XG4gICAgPC9saT5cbiAgKTtcbn07XG5cbmNvbnN0IExhbmd1YWdlUGlja2VyID0gKCkgPT4ge1xuICBjb25zdCBhY3RpdmVMYW5nID0gXCJlblwiO1xuXG4gIHJldHVybiAoXG4gICAgPFN0eWxlZExhbmd1YWdlUGlja2VyPlxuICAgICAgPExhbmd1YWdlU3dpdGNoSWNvbiAvPlxuICAgICAgPHVsPlxuICAgICAgICB7bGFuZ3VhZ2VzLm1hcChsYW5nID0+IChcbiAgICAgICAgICA8TGFuZ3VhZ2VPcHRpb24ga2V5PXtsYW5nfSBsYW5nPXtsYW5nfSBhY3RpdmU9e2xhbmcgPT0gYWN0aXZlTGFuZ30gLz5cbiAgICAgICAgKSl9XG4gICAgICA8L3VsPlxuICAgIDwvU3R5bGVkTGFuZ3VhZ2VQaWNrZXI+XG4gICk7XG59O1xuXG5leHBvcnQgZGVmYXVsdCBMYW5ndWFnZVBpY2tlcjtcbiJdfQ== */;label:LanguageOption;/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImxhbmd1YWdlUGlja2VyLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUE4RFEiLCJmaWxlIjoibGFuZ3VhZ2VQaWNrZXIudHN4Iiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gXCJyZWFjdFwiO1xuaW1wb3J0IHsgY3NzLCBqc3ggfSBmcm9tIFwiQGVtb3Rpb24vY29yZVwiO1xuaW1wb3J0IHN0eWxlZCBmcm9tIFwiQGVtb3Rpb24vc3R5bGVkXCI7XG5pbXBvcnQgeyB1c2VUaGVtZSB9IGZyb20gXCJlbW90aW9uLXRoZW1pbmdcIjtcbmltcG9ydCBMaW5rIGZyb20gXCJnYXRzYnktbGlua1wiO1xuXG5pbXBvcnQgeyBUaGVtZSB9IGZyb20gXCIuLi90aGVtZVwiO1xuXG5pbXBvcnQgeyBMYW5ndWFnZSwgbGFuZ3VhZ2VzLCB0cmFuc2xhdGlvbnMgfSBmcm9tIFwiLi4vLi4vdXRpbHMvdHJhbnNsYXRpb25zXCI7XG5cbmltcG9ydCBMYW5ndWFnZVN3aXRjaEljb24gZnJvbSBcIi4uL2ltYWdlcy9sYW5ndWFnZS1zd2l0Y2hcIjtcblxuY29uc3QgU3R5bGVkTGFuZ3VhZ2VQaWNrZXIgPSBzdHlsZWQuZGl2PHsgdGhlbWU6IFRoZW1lIH0+YFxuICBzdmcge1xuICAgIGZpbGw6ICR7KHsgdGhlbWUgfSkgPT4gdGhlbWUuY29sb3JzLmljb259O1xuICAgIHdpZHRoOiAzMHB4O1xuICB9XG5cbiAgc3ZnOmhvdmVyIHtcbiAgICBmaWxsOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5pY29uSG92ZXJ9O1xuICB9XG5cbiAgdWwge1xuICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgcG9zaXRpb246IGFic29sdXRlO1xuICB9XG5cbiAgOmhvdmVyID4gdWwge1xuICAgIGRpc3BsYXk6IGZsZXg7XG4gICAgZmxleC1kaXJlY3Rpb246IGNvbHVtbjtcbiAgICBiYWNrZ3JvdW5kLWNvbG9yOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5oZWFkZXJCZ307XG4gIH1cbmA7XG5cbmludGVyZmFjZSBMYW5ndWFnZU9wdGlvblByb3BzIHtcbiAgbGFuZzogTGFuZ3VhZ2U7XG4gIGFjdGl2ZTogYm9vbGVhbjtcbn1cblxuY29uc3QgTGFuZ3VhZ2VPcHRpb24gPSAoeyBsYW5nLCBhY3RpdmUgfTogTGFuZ3VhZ2VPcHRpb25Qcm9wcykgPT4ge1xuICBjb25zdCB0aGVtZTogVGhlbWUgPSB1c2VUaGVtZSgpO1xuICBjb25zdCBzdHlsZSA9IGNzc2BcbiAgICBwYWRkaW5nOiAxMHB4IDE1cHg7XG5cbiAgICAmOmhvdmVyIHtcbiAgICAgIGJhY2tncm91bmQ6ICR7dGhlbWUuY29sb3JzLmhpZ2hsaWdodH07XG4gICAgfVxuXG4gICAgYSB7XG4gICAgICBjb2xvcjogJHt0aGVtZS5jb2xvcnMucHJpbWFyeVRleHR9O1xuICAgICAgZm9udC1zaXplOiAxNnB4O1xuICAgICAgZm9udC13ZWlnaHQ6IDUwMDtcbiAgICAgIGxpbmUtaGVpZ2h0OiAxZW07XG4gICAgICBvcGFjaXR5OiAxO1xuICAgIH1cbiAgYDtcblxuICBjb25zdCBhY3RpdmVTdHlsZSA9IGNzc2BcbiAgICBmb250LXdlaWdodDogNzAwO1xuICBgO1xuXG4gIHJldHVybiAoXG4gICAgPGxpIGNzcz17W3N0eWxlLCBhY3RpdmUgJiYgYWN0aXZlU3R5bGVdfT5cbiAgICAgIDxMaW5rIGxhbmc9e2xhbmd9IHRvPVwiL1wiPlxuICAgICAgICB7dHJhbnNsYXRpb25zW2xhbmddW1wibGFuZ3VhZ2VfbmFtZVwiXX1cbiAgICAgIDwvTGluaz5cbiAgICA8L2xpPlxuICApO1xufTtcblxuY29uc3QgTGFuZ3VhZ2VQaWNrZXIgPSAoKSA9PiB7XG4gIGNvbnN0IGFjdGl2ZUxhbmcgPSBcImVuXCI7XG5cbiAgcmV0dXJuIChcbiAgICA8U3R5bGVkTGFuZ3VhZ2VQaWNrZXI+XG4gICAgICA8TGFuZ3VhZ2VTd2l0Y2hJY29uIC8+XG4gICAgICA8dWw+XG4gICAgICAgIHtsYW5ndWFnZXMubWFwKGxhbmcgPT4gKFxuICAgICAgICAgIDxMYW5ndWFnZU9wdGlvbiBrZXk9e2xhbmd9IGxhbmc9e2xhbmd9IGFjdGl2ZT17bGFuZyA9PSBhY3RpdmVMYW5nfSAvPlxuICAgICAgICApKX1cbiAgICAgIDwvdWw+XG4gICAgPC9TdHlsZWRMYW5ndWFnZVBpY2tlcj5cbiAgKTtcbn07XG5cbmV4cG9ydCBkZWZhdWx0IExhbmd1YWdlUGlja2VyO1xuIl19 */",
          "toString": [Function],
        }
      }
    >
      <a
        aria-current="page"
        className=""
        href="/"
        lang="en"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        English
      </a>
    </li>
    <li
      css={
        Object {
          "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImxhbmd1YWdlUGlja2VyLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUF5Q21CIiwiZmlsZSI6Imxhbmd1YWdlUGlja2VyLnRzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tIFwicmVhY3RcIjtcbmltcG9ydCB7IGNzcywganN4IH0gZnJvbSBcIkBlbW90aW9uL2NvcmVcIjtcbmltcG9ydCBzdHlsZWQgZnJvbSBcIkBlbW90aW9uL3N0eWxlZFwiO1xuaW1wb3J0IHsgdXNlVGhlbWUgfSBmcm9tIFwiZW1vdGlvbi10aGVtaW5nXCI7XG5pbXBvcnQgTGluayBmcm9tIFwiZ2F0c2J5LWxpbmtcIjtcblxuaW1wb3J0IHsgVGhlbWUgfSBmcm9tIFwiLi4vdGhlbWVcIjtcblxuaW1wb3J0IHsgTGFuZ3VhZ2UsIGxhbmd1YWdlcywgdHJhbnNsYXRpb25zIH0gZnJvbSBcIi4uLy4uL3V0aWxzL3RyYW5zbGF0aW9uc1wiO1xuXG5pbXBvcnQgTGFuZ3VhZ2VTd2l0Y2hJY29uIGZyb20gXCIuLi9pbWFnZXMvbGFuZ3VhZ2Utc3dpdGNoXCI7XG5cbmNvbnN0IFN0eWxlZExhbmd1YWdlUGlja2VyID0gc3R5bGVkLmRpdjx7IHRoZW1lOiBUaGVtZSB9PmBcbiAgc3ZnIHtcbiAgICBmaWxsOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5pY29ufTtcbiAgICB3aWR0aDogMzBweDtcbiAgfVxuXG4gIHN2Zzpob3ZlciB7XG4gICAgZmlsbDogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaWNvbkhvdmVyfTtcbiAgfVxuXG4gIHVsIHtcbiAgICBkaXNwbGF5OiBub25lO1xuICAgIHBvc2l0aW9uOiBhYnNvbHV0ZTtcbiAgfVxuXG4gIDpob3ZlciA+IHVsIHtcbiAgICBkaXNwbGF5OiBmbGV4O1xuICAgIGZsZXgtZGlyZWN0aW9uOiBjb2x1bW47XG4gICAgYmFja2dyb3VuZC1jb2xvcjogJHsoeyB0aGVtZSB9KSA9PiB0aGVtZS5jb2xvcnMuaGVhZGVyQmd9O1xuICB9XG5gO1xuXG5pbnRlcmZhY2UgTGFuZ3VhZ2VPcHRpb25Qcm9wcyB7XG4gIGxhbmc6IExhbmd1YWdlO1xuICBhY3RpdmU6IGJvb2xlYW47XG59XG5cbmNvbnN0IExhbmd1YWdlT3B0aW9uID0gKHsgbGFuZywgYWN0aXZlIH06IExhbmd1YWdlT3B0aW9uUHJvcHMpID0+IHtcbiAgY29uc3QgdGhlbWU6IFRoZW1lID0gdXNlVGhlbWUoKTtcbiAgY29uc3Qgc3R5bGUgPSBjc3NgXG4gICAgcGFkZGluZzogMTBweCAxNXB4O1xuXG4gICAgJjpob3ZlciB7XG4gICAgICBiYWNrZ3JvdW5kOiAke3RoZW1lLmNvbG9ycy5oaWdobGlnaHR9O1xuICAgIH1cblxuICAgIGEge1xuICAgICAgY29sb3I6ICR7dGhlbWUuY29sb3JzLnByaW1hcnlUZXh0fTtcbiAgICAgIGZvbnQtc2l6ZTogMTZweDtcbiAgICAgIGZvbnQtd2VpZ2h0OiA1MDA7XG4gICAgICBsaW5lLWhlaWdodDogMWVtO1xuICAgICAgb3BhY2l0eTogMTtcbiAgICB9XG4gIGA7XG5cbiAgY29uc3QgYWN0aXZlU3R5bGUgPSBjc3NgXG4gICAgZm9udC13ZWlnaHQ6IDcwMDtcbiAgYDtcblxuICByZXR1cm4gKFxuICAgIDxsaSBjc3M9e1tzdHlsZSwgYWN0aXZlICYmIGFjdGl2ZVN0eWxlXX0+XG4gICAgICA8TGluayBsYW5nPXtsYW5nfSB0bz1cIi9cIj5cbiAgICAgICAge3RyYW5zbGF0aW9uc1tsYW5nXVtcImxhbmd1YWdlX25hbWVcIl19XG4gICAgICA8L0xpbms+XG4gICAgPC9saT5cbiAgKTtcbn07XG5cbmNvbnN0IExhbmd1YWdlUGlja2VyID0gKCkgPT4ge1xuICBjb25zdCBhY3RpdmVMYW5nID0gXCJlblwiO1xuXG4gIHJldHVybiAoXG4gICAgPFN0eWxlZExhbmd1YWdlUGlja2VyPlxuICAgICAgPExhbmd1YWdlU3dpdGNoSWNvbiAvPlxuICAgICAgPHVsPlxuICAgICAgICB7bGFuZ3VhZ2VzLm1hcChsYW5nID0+IChcbiAgICAgICAgICA8TGFuZ3VhZ2VPcHRpb24ga2V5PXtsYW5nfSBsYW5nPXtsYW5nfSBhY3RpdmU9e2xhbmcgPT0gYWN0aXZlTGFuZ30gLz5cbiAgICAgICAgKSl9XG4gICAgICA8L3VsPlxuICAgIDwvU3R5bGVkTGFuZ3VhZ2VQaWNrZXI+XG4gICk7XG59O1xuXG5leHBvcnQgZGVmYXVsdCBMYW5ndWFnZVBpY2tlcjtcbiJdfQ== */",
          "name": "ii8b9y-style-LanguageOption",
          "next": undefined,
          "styles": "padding:10px 15px;&:hover{background:#3884C7;}a{color:#3B454E;font-size:16px;font-weight:500;line-height:1em;opacity:1;};label:style;;;label:LanguageOption;/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImxhbmd1YWdlUGlja2VyLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUE4RFEiLCJmaWxlIjoibGFuZ3VhZ2VQaWNrZXIudHN4Iiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gXCJyZWFjdFwiO1xuaW1wb3J0IHsgY3NzLCBqc3ggfSBmcm9tIFwiQGVtb3Rpb24vY29yZVwiO1xuaW1wb3J0IHN0eWxlZCBmcm9tIFwiQGVtb3Rpb24vc3R5bGVkXCI7XG5pbXBvcnQgeyB1c2VUaGVtZSB9IGZyb20gXCJlbW90aW9uLXRoZW1pbmdcIjtcbmltcG9ydCBMaW5rIGZyb20gXCJnYXRzYnktbGlua1wiO1xuXG5pbXBvcnQgeyBUaGVtZSB9IGZyb20gXCIuLi90aGVtZVwiO1xuXG5pbXBvcnQgeyBMYW5ndWFnZSwgbGFuZ3VhZ2VzLCB0cmFuc2xhdGlvbnMgfSBmcm9tIFwiLi4vLi4vdXRpbHMvdHJhbnNsYXRpb25zXCI7XG5cbmltcG9ydCBMYW5ndWFnZVN3aXRjaEljb24gZnJvbSBcIi4uL2ltYWdlcy9sYW5ndWFnZS1zd2l0Y2hcIjtcblxuY29uc3QgU3R5bGVkTGFuZ3VhZ2VQaWNrZXIgPSBzdHlsZWQuZGl2PHsgdGhlbWU6IFRoZW1lIH0+YFxuICBzdmcge1xuICAgIGZpbGw6ICR7KHsgdGhlbWUgfSkgPT4gdGhlbWUuY29sb3JzLmljb259O1xuICAgIHdpZHRoOiAzMHB4O1xuICB9XG5cbiAgc3ZnOmhvdmVyIHtcbiAgICBmaWxsOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5pY29uSG92ZXJ9O1xuICB9XG5cbiAgdWwge1xuICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgcG9zaXRpb246IGFic29sdXRlO1xuICB9XG5cbiAgOmhvdmVyID4gdWwge1xuICAgIGRpc3BsYXk6IGZsZXg7XG4gICAgZmxleC1kaXJlY3Rpb246IGNvbHVtbjtcbiAgICBiYWNrZ3JvdW5kLWNvbG9yOiAkeyh7IHRoZW1lIH0pID0+IHRoZW1lLmNvbG9ycy5oZWFkZXJCZ307XG4gIH1cbmA7XG5cbmludGVyZmFjZSBMYW5ndWFnZU9wdGlvblByb3BzIHtcbiAgbGFuZzogTGFuZ3VhZ2U7XG4gIGFjdGl2ZTogYm9vbGVhbjtcbn1cblxuY29uc3QgTGFuZ3VhZ2VPcHRpb24gPSAoeyBsYW5nLCBhY3RpdmUgfTogTGFuZ3VhZ2VPcHRpb25Qcm9wcykgPT4ge1xuICBjb25zdCB0aGVtZTogVGhlbWUgPSB1c2VUaGVtZSgpO1xuICBjb25zdCBzdHlsZSA9IGNzc2BcbiAgICBwYWRkaW5nOiAxMHB4IDE1cHg7XG5cbiAgICAmOmhvdmVyIHtcbiAgICAgIGJhY2tncm91bmQ6ICR7dGhlbWUuY29sb3JzLmhpZ2hsaWdodH07XG4gICAgfVxuXG4gICAgYSB7XG4gICAgICBjb2xvcjogJHt0aGVtZS5jb2xvcnMucHJpbWFyeVRleHR9O1xuICAgICAgZm9udC1zaXplOiAxNnB4O1xuICAgICAgZm9udC13ZWlnaHQ6IDUwMDtcbiAgICAgIGxpbmUtaGVpZ2h0OiAxZW07XG4gICAgICBvcGFjaXR5OiAxO1xuICAgIH1cbiAgYDtcblxuICBjb25zdCBhY3RpdmVTdHlsZSA9IGNzc2BcbiAgICBmb250LXdlaWdodDogNzAwO1xuICBgO1xuXG4gIHJldHVybiAoXG4gICAgPGxpIGNzcz17W3N0eWxlLCBhY3RpdmUgJiYgYWN0aXZlU3R5bGVdfT5cbiAgICAgIDxMaW5rIGxhbmc9e2xhbmd9IHRvPVwiL1wiPlxuICAgICAgICB7dHJhbnNsYXRpb25zW2xhbmddW1wibGFuZ3VhZ2VfbmFtZVwiXX1cbiAgICAgIDwvTGluaz5cbiAgICA8L2xpPlxuICApO1xufTtcblxuY29uc3QgTGFuZ3VhZ2VQaWNrZXIgPSAoKSA9PiB7XG4gIGNvbnN0IGFjdGl2ZUxhbmcgPSBcImVuXCI7XG5cbiAgcmV0dXJuIChcbiAgICA8U3R5bGVkTGFuZ3VhZ2VQaWNrZXI+XG4gICAgICA8TGFuZ3VhZ2VTd2l0Y2hJY29uIC8+XG4gICAgICA8dWw+XG4gICAgICAgIHtsYW5ndWFnZXMubWFwKGxhbmcgPT4gKFxuICAgICAgICAgIDxMYW5ndWFnZU9wdGlvbiBrZXk9e2xhbmd9IGxhbmc9e2xhbmd9IGFjdGl2ZT17bGFuZyA9PSBhY3RpdmVMYW5nfSAvPlxuICAgICAgICApKX1cbiAgICAgIDwvdWw+XG4gICAgPC9TdHlsZWRMYW5ndWFnZVBpY2tlcj5cbiAgKTtcbn07XG5cbmV4cG9ydCBkZWZhdWx0IExhbmd1YWdlUGlja2VyO1xuIl19 */",
          "toString": [Function],
        }
      }
    >
      <a
        aria-current="page"
        className=""
        href="/"
        lang="zh-hans"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        简体中文
      </a>
    </li>
  </ul>
</div>
`;
